### PR TITLE
Adapt calculator to design

### PR DESCRIPTION
**Pull request by Said-laasri**
**React Calculator:**

Deleted all the boilerplate from CRA (text, images, styles).
Created a directory called `components.`
 created a new `Calculator.js`  Inside `components, file.`
In `Calculator.js,` created a React component that matches the design.
Used class-based components

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    return (
+      <div className="calculator">
+        <div className="answer">
+          <span>0</span>
+        </div>
+        <div className="section1">
+          <button type="button" className="number">AC</button>
+          <button type="button" className="number">+/-</button>
+          <button type="button" className="number">%</button>
+          <button type="button" className="side-symbols">+</button>
```
Comment (Mre55) [inline src/components/Calculator.js]: Make sure to have division operator which if you want you can use `/`. You have 2 `+` buttons at line 14 and line 16 make sure to match the design provided.

| Your Design | Provided design |
|-------------|---------------|
| image | image |

**Review by Mre55 — CHANGES_REQUESTED** [review]
**Hi @Said-laasri,**

You have done a great job so far! 💪 👏 
You still need to work on a few issues before moving on to the next project, but you're almost there!

## To highlight 🎯

- The Pull Request is following the Git Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 

## Required Changes ♻️🛠️ 

- [x] Make sure all the boilerplate that got created by the CRA (texts, images, and styles) ought to have gotten deleted. You have some inside `public` folder.

_Check the comments under the review._




## Optional suggestions 🟡 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (Said-laasri) [reply src/components/Calculator.js]: @Mre55   Thank you for reviewing my project, I will make the necessary corrections as suggested

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @Said-laasri,

✔️ App design matches wireframe

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Said-laasri** [comment]
@TedLivist  Thank you for reviewing my project.

**Pull request merged**